tests/excerptexport/models/test_export_model.py

```diff
@@ -0,0 +1,31 @@
+from datetime import timedelta
+
+import pytest
+
+from django.utils.datetime_safe import datetime
+
+
+@pytest.mark.django_db
+def test_created_export_has_correct_time_stamp(export):
+    now = datetime.now()
+    margin = timedelta(minutes=1)
+    assert export.finished is None
+    assert export.updated is not None
+    assert export.created is not None
+    assert export.created == export.updated
+    assert (now - margin) < export.created < (now + margin)
```

Comment:
Could also be written as
```python
assert abs(export.created - now) < margin
```
but explicit bounds are fine, too.

Could this be tested without margin by mocking out `datetime.now()`?